### ISSUE TITLE
Feedback feed form lacks label

### ISSUE THREAD

**Issue by torgeirl** [devilry/devilry-django#1211]
WCAG 2.1 Success Criterion 4.1.1 requires content implemented using markup languages, elements have complete start and end tags, elements are nested according to their specifications, elements do not contain duplicate attributes, and any IDs are unique, except where the specifications allow these features.

The `<form>` element in the feedback feed lacks a label, making it hard to understand when navigating using assistance tools.

**Comment by StianJul** [comment]
@torgeirl, we've looked into the issue and have done some digging regarding a label for the textarea. It is _kind of_ possible to achieve, but it will not make the ace-editor anymore useful for screenreaders in most cases. It seems that going down this road will only result in a, at best, subpar solution.

https://github.com/ajaxorg/ace/issues/2164

We should instead implement a new text-editor based on a standard textarea to make it more accessibility-friendly, such as the editor used by GitHub.

**Comment by torgeirl** [comment]
> We should instead implement a new text-editor based on a standard textarea to make it more accessibility-friendly, such as the editor used by GitHub.

@StianJul: that sounds like a much better solution, and should also be a step towards solving some of the other issues with the editor (#917, #975, #994, #1040, #1174).

**Comment by StianJul** [comment]
https://github.com/devilry/devilry-django/issues/1217

**Comment by torgeirl** [comment]
> Will be solved with #1217

Same as #1210, I assume this therefore is solved as part of #1214?
